eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,188 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    # Download package from nuget, if no package found in nuget, then search devops public feeds if set.
+    Write-Host "Downloading $package with version $version..."
+    if ($customPackageSource) {
+        nuget install -FallbackSource $customPackageSource $package -Version $version -DirectDownload -DependencyVersion Ignore -OutputDirectory $destination
+    }
+    else {
+        nuget install $package -Version $version -DirectDownload -DependencyVersion Ignore -OutputDirectory $destination
+    }
+}
+
+function Fetch-NamespacesFromNupkg ($package, $version) {
+    $tempLocation = (Join-Path ([System.IO.Path]::GetTempPath()) "extractNupkg")
+    if (!(Test-Path $tempLocation)) {
+        New-Item -ItemType Directory -Path $tempLocation -Force | Out-Null
+    }
+    Write-Host "Downloading nupkg package $package with version $version to $tempLocation ...."
+    DownloadNugetPackage -package $package -version $version -destination $tempLocation | Out-Null
+    $packageFolder = "$package.$version"
+    if ($LASTEXITCODE -ne 0 -or !$(Test-Path $tempLocation/$packageFolder)) {
+        Write-Host "Did not download the package correctly. Skipping..."
+        return @()
+    }
+    Write-Host "Searching for dll file..."
+    $dllFile = @()
+    if (Test-Path "$tempLocation/$packageFolder/lib/netstandard2.0/"){
+        $dllFile = @(Get-ChildItem "$tempLocation/$packageFolder/lib/netstandard2.0/*" -Filter '*.dll' -Recurse)
+    }
+    if (!$dllFile) {
+        Write-Warning "Can't find any dll file from $tempLocation/$packageFolder with target netstandard2.0."
+        $dllFiles = Get-ChildItem "$tempLocation/$packageFolder/lib/*" -Filter '*.dll' -Recurse
+        if (!$dllFiles) {
+            Write-Error "Can't find any dll file from $tempLocation/$packageFolder." -ErrorAction Continue
+            return @()
+        }
+        $dllFile = $dllFiles[0]
+    }
+    elseif ($dllFile.Count -gt 1) {
+        Write-Warning "There are multiple dll files in target netstandard2.0. Choose the one with package name."
```

Comment:
You should move this warning after you select the dll so you can include the name of the dll in the output.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/32062/files#diff-4b590925689e1df66d5c28ddc67d15ded4db6d686eb7c7878e0c8238feb57e25R76

I have printed at end. The message here shows the route to find dll